packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -262,10 +264,13 @@ def get_nodes_from_response(response):
 
         agent_nodes_response = check_json(
             dcos_api_session.health.get('/units/dcos-mesos-slave.service/nodes', node=master))
-
         agent_nodes = get_nodes_from_response(agent_nodes_response)
 
-        assert len(agent_nodes) == len(dcos_api_session.slaves), '{} != {}'.format(agent_nodes, dcos_api_session.slaves)
+        windows_agent_nodes_response = check_json(
+            dcos_api_session.health.get('/units/mesos-agent/nodes', node=master))
```

Comment:
@DominikDary and @janisz, this points to inconsistencies between Linux and Windows agents. Ideally the service names would be the same or at least indicate that one is a Linux agent and the other a Windows agent. Do we have a PO for the Windows support?